ts/python/mod.ts

```diff
@@ -0,0 +1,114 @@
+import { Environment, mkEnvironment } from "../environment.ts";
+import { mkPackage, Package } from "../package.ts";
+import { mkProject, Project } from "../project.ts";
+import { nixSource } from "../internal/utils.ts";
+import { nixRaw } from "../nix.ts";
+import * as pythonInterpreters from "../internal/nixpkgs/pythonInterpreters/mod.ts";
+import { parsePyprojectToml } from "./utils.ts";
+export { plugin as vite } from "../javascript/vite.ts";
+
+export const interpreters = pythonInterpreters;
+
+/**
+ * Creates a pyproject based python garn Project.
```

Comment:
Maybe a bit more documentation would be nice, especially a `@param` for pythonIntepreter. `mkHaskellProject` is a good inspiration (the other ones not so much)

Reply:
fixed via 56c2e3245609dc7cdd46ded683de46e33c3568fd